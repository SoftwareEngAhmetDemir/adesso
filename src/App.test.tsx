import { render, screen } from '@testing-library/react';
import App from './App';
import { shallow } from 'enzyme';


test('renders App', () => {
    const wrapper = shallow(<App/>);
    expect(wrapper).toMatchSnapshot();
});