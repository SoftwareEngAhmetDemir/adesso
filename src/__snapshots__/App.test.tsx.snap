// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `
<div
  className="App"
>
  <Router />
</div>
`;
